cunumeric/module.py

```diff
@@ -3009,6 +3009,64 @@ def einsum(expr, *operands, out=None, optimize=False):
     return operands[0]
 
 
+@add_boilerplate("a", "out")
```

Comment:
`out` is handled automatically, you don't need to include it in `add_boilerplate`